qiskit/transpiler/preset_passmanagers/builtin_plugins.py

```diff
@@ -1003,3 +1021,63 @@ def _get_trial_count(default_trials=5):
     if CONFIG.get("sabre_all_threads", None) or os.getenv("QISKIT_SABRE_ALL_THREADS"):
         return max(default_num_processes(), default_trials)
     return default_trials
+
+
+class CliffordTOptimizationPassManager(PassManagerStagePlugin):
```

Comment:
Do you use somewhere the `CollectClliffords` transpiler pass? I think it can collect parametrized gates that are clifford only for certain parameters.